ddl/table.go

```diff
@@ -74,6 +74,7 @@ func onCreateTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 			// TODO: Add restrictions to this operation.
 			go splitTableRegion(d.store, tbInfo.ID)
```

Comment:
Should we modify splitTableRegion strategy here?

Reply:
This PR is not to do this.